homeassistant/components/airzone/__init__.py

```diff
@@ -0,0 +1,64 @@
+"""The Airzone integration."""
+from __future__ import annotations
+
+from aioairzone.common import ConnectionOptions
+from aioairzone.localapi_device import AirzoneLocalApi
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import AirzoneUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+class AirzoneEntity(CoordinatorEntity):
+    """Define an Airzone entity."""
+
+    def __init__(
+        self,
+        coordinator: AirzoneUpdateCoordinator,
+        zone_id: str,
+        zone_name: str,
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator)
+
+        self._attr_device_info: DeviceInfo = {
+            "identifiers": {(DOMAIN, zone_id)},
```

Comment:
`zone_id` should probably be the unique id

Reply:
> `zone_id` should probably be the unique id

If we do that, and one user has the following two devices, wouldn't it be a problem?
Device A (192.168.1.10:3000): System: 1, Zones: 1, 2, 3
Device B (192.168.1.20:3000): System 1, Zones: 1, 2

Reply:
Maybe use `(DOMAIN, {entry_id}_{system_id}_{zone_id})`?

Reply:
revised suggestion above

Reply:
Changed in https://github.com/home-assistant/core/commit/18681e4e4cea7e9d8fb226eb3ff9fa963507ad1e